README.md

```diff
@@ -293,10 +287,25 @@ installed.
 1. [Build](#build) the changes and push them to some tag
 2. Use the new tag in your workflows:
 
-```diff
--uses: paritytech/pr-custom-review@1
-+uses: paritytech/pr-custom-review@2
-```
+    ```diff
+    -uses: paritytech/pr-custom-review@1
+    +uses: paritytech/pr-custom-review@2
+    ```
+
+### Deployment
+
+1. Create a token with `workflow`, `read:org`, `user` scopes.
+
+    ![Token scopes](./img/token-scopes.png)
+
+    and save it into the repo or orga secrets.
+
+    Use this token in `with: token:` of the Action definition.
+
+2. one should [add the config](https://github.com/paritytech/substrate/pull/10968) to the repository, without it in `master` the Action will not work.
```

Comment:
https://github.com/paritytech/pr-custom-review/pull/61

https://github.com/paritytech/pr-custom-review/blob/e52ae0a982c6b3c3f89336b2ceb5ca2e2b34bb35/README.md?plain=1#L61

This whole step should just be "Add the configuration file" with a link to the "Action configuration" section